Makefile

```
@@ -1,9 +1,9 @@
 all: lint test demo
 
-GO_BUILD_TAGS:=no_net,no_json,no_pprof
+GO_BUILD_TAGS?=no_net,no_json,no_pprof
```

```suggestion
GO_BUILD_TAGS:=no_net,no_json,no_pprof
```
reverting this, I want `make demo GO_BUILD_TAGS=foo` to change the tags, not using env var